actor/builtin/miner/miner.go

```diff
@@ -553,7 +553,7 @@ func (ma *Actor) CommitSector(ctx exec.VMContext, sectorID uint64, commD, commR,
 		state.ActiveCollateral = state.ActiveCollateral.Add(collateral)
 
 		if state.Power.Equal(types.NewBytesAmount(0)) {
-			state.ProvingPeriodStart = ctx.BlockHeight()
+			state.ProvingPeriodEnd = ctx.BlockHeight()
```

Comment:
Must add proving period duration to block height to compute the end (copy from 742)